docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md

```diff
@@ -125,11 +125,68 @@ will result in the following JSON structure:
 
 This section describes how the HTTP Request tag will use the context Entities attached to a Snowplow Event.
 
+![snowplow event context rules](images/context_rules.png)
+
 #### Extract entity from Array if single element
 
 Snowplow Entities are always in Arrays, as multiple of the same entity can be attached to an event. This option will pick the single element from the array if the array only contains a single element.
 
-#### Include all Entities in request body
+#### Include Snowplow Entities in request body
+
+Using this drop-down menu you can specify whether you want to Include `All` (default) or `None` of the Snowplow context entities in HTTP Request's body.
+
+#### Nest all unmapped Entities under key
+
+This option is available only if the previous option ([Include Snowplow Entities in request body](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md#include-snowplow-entities-in-request-body)) is set to `All`.
+
+It applies **only** to unmapped entities, i.e. all included entities whose mapping is not edited in the following ([Snowplow Entities to Add/Edit mapping](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md#snowplow-entities-to-addedit-mapping)) table.
+
+With this setting you can specify a key under which the Snowplow event's unmapped entities will be nested. Alternatively, leaving it blank adds the unmapped entities in the request body without nesting. You can also use dot notation in this value.
+
+#### Snowplow Entities to Add/Edit mapping
+
+Using this table you can specify in each row a specific mapping for a particular context entity. In the columns provided you can specify:
+
+- **Entity Name**: The Entity name to add/edit-mapping (required).¹
+- **Destination Mapped Name**: The key you could like to map it to in the request body (optional: leaving the mapped key blank keeps the same name). You can use dot notation here as well to signify further nesting. This value is independent of the [nesting of unmapped entities](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md#nest-all-unmapped-entities-under-key) setting above.
```

Comment:
for links within the same page you can just use `[...](#nest-all-unmapped-entities-under-key)`